### PR TITLE
Jest codemods preparation

### PR DESCRIPTION
**Pull request by skovhus**
This is the first part of migrating to Jest:
- remove commented out tests (Jest doesn't like these, and they can always be removed from git if you need them)
- fix tests that contains no assertions (Jest found this...)

The rest of the migration to Jest should be done by [`jest-codemods`](https://github.com/skovhus/jest-codemods), I'm working on that.

### CODE REVIEW

**Comment by skovhus** [comment]
@erikras a bit of preparation that can be merged before I'm migrating to Jest.

File: src/__tests__/createFieldArrayProps.spec.js [hunk 1]
```diff
@@ -278,7 +278,7 @@ const describeCreateFieldProps = (name, structure, expect) => {
       expect(result.fields.map).toBeA('function')
       expect(callback).toNotHaveBeenCalled()
       const mapResult = result.fields.map(callback)
-      expect(size(mapResult), 3)
```
Comment (skovhus) [inline src/__tests__/createFieldArrayProps.spec.js]: Should we just remove these? For some reason `size(mapResult)` returns `undefined`.

`Error: Expected undefined to be 3`

https://travis-ci.org/erikras/redux-form/jobs/273908404

**Comment by erikras** [comment]
Does this negate all the work I put into the `jest` branch?

**Comment by skovhus** [comment]
Here I'm going for automating most of the changes, so the next person doesn't need all that work. 

I might be that some of the changes you did Anine transferred. : )

**Comment by skovhus** [comment]
Replaced by https://github.com/erikras/redux-form/pull/3413/

**Pull request closed without merging**